src/utils/normalization.ts

```diff
@@ -15,8 +14,8 @@ export const normalize = (payload: OperationsRepository) => {
 
       const beacons = Object.fromEntries(
         Object.entries(api.beacons).map(([_key, beacon]) => {
-          const beaconId = keccak256(
-            defaultAbiCoder.encode(['address', 'bytes32'], [beacon.airnodeAddress, beacon.templateId])
+          const beaconId = ethers.utils.keccak256(
+            ethers.utils.defaultAbiCoder.encode(['address', 'bytes32'], [beacon.airnodeAddress, beacon.templateId])
```

Comment:
At the risk of causing confusion :sweat_smile:, we are deriving **beaconIds** like this in Airkeeper:
```
const beaconId = ethers.utils.keccak256(
    ethers.utils.solidityPack(['address', 'bytes32'], [airnodeAddress, templateId])
  );
```
These two produce different outputs :thinking:

Reply:
Wow, does these two really produce different versions? I thought that they are the same functions just imported from different packages. If they are really different we should check all repos (also private) and make this consistent.

Reply:
I can confirm that @aquarat 's change doesn't change the output. @vponline 's change does change the beaconIds in the existing repo as well, so I don't know how our way matches the output in data/beacons folder